app/controllers/tweets.rb

```diff
@@ -0,0 +1,24 @@
+get '/tweets/new' do
+  @tweet = Tweet.new
+  erb :"/tweets/new"
+end
+
+post '/tweets' do
+  @tweet = Tweet.new(user_id: current_user.id, content: params[:tweet][:content])
+  if @tweet.save
+    redirect "/users/current_user.id"
+  else
+    @tweet.errors.full_messages
+    erb :"/tweets/new"
+  end
+end
+
+delete '/tweets' do
+  @tweet = Tweet.find_by(user_id: current_user.id)
```

Comment:
This would return multiple tweets, so you would be destroying all of the user's tweets.